platform/tezos/model.go

```diff
@@ -67,16 +67,18 @@ func (t *Transaction) ErrorMsg() string {
 	}
 }
 
-func (t *Transaction) Title() blockatlas.KeyTitle {
-	if t.Delegate == "" && t.Receiver != "" {
-		return blockatlas.AnyActionDelegation
+func (t *Transaction) Title(address string) blockatlas.KeyTitle {
```

Comment:
does not seem unintuitive to pass address to Title function

Reply:
So far only one way i see how we can determine type